src/Microsoft.Health.Fhir.TemplateManagement/Microsoft.Health.Fhir.TemplateManagement.csproj

```diff
@@ -52,36 +52,42 @@
       <Link>JsonDefaultTemplates.tar.gz</Link>
     </EmbeddedResource>
   </ItemGroup>
-	
+
   <ItemGroup>
     <None Update="oras.exe">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-    
+
   <ItemGroup>
     <None Update="oras-OSX">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  
+
+  <ItemGroup>
+    <None Include="$(BinFolder)oras_$(OrasVersion)_darwin_amd64.tar.gz">
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
+    
   <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
     <Exec Command="mkdir $(BinFolder) &amp; tar -zcvf $(BinFolder)Hl7v2DefaultTemplates.tar.gz -C $(DataFolder)Templates/Hl7v2 ." />
     <Exec Command="tar -zcvf $(BinFolder)CcdaDefaultTemplates.tar.gz -C $(DataFolder)Templates/Ccda ." />
     <Exec Command="tar -zcvf $(BinFolder)JsonDefaultTemplates.tar.gz -C $(DataFolder)Templates/Json ." />
   </Target>
-  
+
   <Target Name="DownloadWinOrasFile" BeforeTargets="Build">
     <DownloadFile SourceUrl="$(OrasWinUrl)" DestinationFolder="$(BinFolder)">
       <Output TaskParameter="DownloadedFile" ItemName="Content" />
     </DownloadFile>
     <Exec Command="tar -xvf $(BinFolder)oras_$(OrasVersion)_windows_amd64.tar.gz -C $(BinFolder) &amp; $(CopyExecutable) $(BinFolder)oras.exe ." />
```

Comment:
Same as MacOS package name. oras_$(OrasVersion)_windows_amd64.tar.gz can be a variable.